docs/tutorials/zarr-v3.md

```diff
@@ -0,0 +1,106 @@
+
+## Codecs
+
+The default `zarr-python` v3 codec for the v3 format is no longer `blosc` but `zstd`.
+While `zstd` is more widespread, you may find its performance to not meet your old expectations.
+Therefore, we recommend passing in the [`BloscCodec`] to `compressor` on {func}`~anndata.AnnData.write_zarr` if you wish to return to the old behavior.
+
+There is currently a bug with `numcodecs` that prevents data written from other non-numcodecs `zstd` implementations from being read in by the default zarr pipeline (to which the above rust pipeline falls back if it cannot handle a datatype or indexing scheme, like `vlen-string`): {issue}`zarr-developers/numcodecs#424`.
+Thus is may be advisable to use `BloscCodec` with `zarr` v3 file format data if you wish to use the rust-accelerated pipeline until this issue is resolved.
+
+The same issue with `zstd` applies to data that may eventually be written by the GPU `zstd` implementation (see below).
+
+## GPU i/o
+
+At the moment, it is unlikely your `anndata` i/o will work if you use [`zarr.enable_gpu`].
+It's *possible* dense data i/o i.e., using {func}`anndata.io.read_elem` will work as expected, but this functionality is untested – sparse data, awkward arrays, and dataframes will not.
+`kvikio` currently provides a [`GDS`-enabled store] although there are no working compressors at the moment exported from the `zarr-python` package (work is underway for `Zstd`: {pr}`zarr-developers/zarr-python#2863`.
+
+We anticipate enabling officially supporting this functionality officially for dense data, sparse data, and possibly awkward arrays in the next minor release, 0.13.
+
+## Asynchronous i/o
+
+At the moment, `anndata` exports no `async` functions.
+However, `zarr-python` has a fully `async` API and provides its own event-loop so that users like `anndata` can interact with a synchronous API while still beenfitting from `zarr-python`'s asynchronous functionality under that API.
+We anticipate providing `async` versions of {func}`anndata.io.read_elem` and {func}`anndata.experimental.read_dispatched` so that users can download data asynchronously without using the `zarr-python` event loop.
+We also would like to create an asynchronous partial reader to enable iterative streaming of a dataset.
+
+[consolidated metadata]: https://zarr.readthedocs.io/en/stable/user-guide/consolidated_metadata.html
+[`zarr` stores]: https://zarr.readthedocs.io/en/stable/api/zarr/storage/index.html
+[`obstore` claims]: https://developmentseed.org/obstore/latest/performance
+[sharding]: https://zarr.readthedocs.io/en/stable/user-guide/performance.html#sharding
+[`BloscCodec`]: https://zarr.readthedocs.io/en/stable/api/zarr/codecs/index.html#zarr.codecs.BloscCodec
+[`zarr.enable_gpu`]: https://zarr.readthedocs.io/en/stable/user-guide/gpu.html#reading-data-into-device-memory
+[`GDS`-enabled store]: https://docs.rapids.ai/api/kvikio/nightly/api/#kvikio.zarr.GDSStore
```

Comment:
Link all of these using roles, not markdown links, as

- https://docs.rapids.ai/api/kvikio/nightly/objects.inv exists
- https://zarr.readthedocs.io/en/stable/objects.inv exists
- https://developmentseed.org/obstore/latest/objects.inv exists

So please update the intersphinx config as appropriate, find all these objects using e.g. the `sphobjinv` tool, and link them without URLs.

```suggestion
```